packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -186,6 +202,21 @@ export class VirtualGateway extends VirtualGatewayBase {
       resourceName: this.physicalName,
     });
   }
+
+  /**
+   * Adds Default Backend Configuration for Virtual Gateway to communicate with Virtual Services.
+   */
+  private addBackendDefaults(clientPolicy: ClientPolicy): CfnVirtualGateway.VirtualGatewayBackendDefaultsProperty {
+    return {
+      clientPolicy: {
+        tls: {
+          enforce: clientPolicy.tlsClientPolicy.enforce ?? true,
+          validation: clientPolicy.tlsClientPolicy.validation.bind(this).tlsValidation,
```

Comment:
Why is the `tslValidation` set in `bind()`, but the rest are properties of the `ClientPolicy`? Wondering why the different approach.

Reply:
Earlier, I was trying to separate out the validation context as it could be used in future for Listener TLS.
But now I have a single file `client-policy.ts` and now the `clientPolicy` properties are being set in `bind()`.

This would make the API more intuitive.

```typescript
new appmesh.VirtualGateway(stack, 'virtual-gateway', {
 virtualGatewayName: 'virtual-gateway',
 mesh: mesh,
 backendDefaults: appmesh.ClientPolicy.fileTrust({
   certificateChain: ['path-to-certificate'],
   enforceTls: false,
 }),
});
```